test/k8sT/KafkaPolicies.go

```diff
@@ -57,9 +60,30 @@ var _ = Describe("K8sValidatedKafkaPolicyTest", func() {
 			"cilium endpoint list")
 	})
 
-	// GH-4414: put test in a context so that if any failures occur in BeforeAll,
-	// logs will be gathered by the above "AfterFailed".
 	Context("Kafka Policy Tests", func() {
+		createTopicCmd := func(topic string) string {
+			return fmt.Sprintf("/opt/kafka/bin/kafka-topics.sh --create --zookeeper zook:2181 "+
+				"--replication-factor 1 --partitions 1 --topic %s", topic)
+		}
+
+		createTopic := func(topic string, pod string) error {
+			return kubectl.ExecKafkaPodCmd(helpers.DefaultNamespace, pod, createTopicCmd(topic))
+		}
+
+		// WaitKafkaBroker waits for the broker to be ready, by executing
+		// a produce request on existing topics and waiting for a response from broker
```

Comment:
`a produce request on existing topics and waiting for a response from broker` -> `a command repeatedly until it succeeds, or a timeout occurs`

Reply:
done. Also changed wording in `runtime` tests